Counterexamples/AharoniKorman.lean

```diff
@@ -0,0 +1,325 @@
+/-
+Copyright (c) 2024 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+
+import Mathlib.Algebra.Order.Star.Basic
+import Mathlib.Data.Nat.Cast.Order.Ring
+import Mathlib.Order.Chain
+import Mathlib.Order.WellFoundedSet
+import Mathlib.Data.Setoid.Partition
+
+/-!
+# Disproof of the Aharoni-Korman conjecture
```

Comment:
I am aware that this suggestion is more likely to make you think "wow, that Mitchell sure is an insufferable pedant" than "thanks, Mitchell!". I hope you can forgive me.
```suggestion
# Disproof of the Aharoni–Korman conjecture
```